s3/main.tf

```diff
@@ -351,17 +364,19 @@ resource "aws_kms_key_policy" "s3-ssm-kms-key-policy" {
 }
 
 resource "aws_ssm_parameter" "cloudfront_alias" {
-  count  = var.config.serve_static_content ? 1 : 0
+  count = var.config.serve_static_content ? 1 : 0
+
   name   = "/copilot/${var.application}/${var.environment}/secrets/STATIC_S3_ENDPOINT"
   type   = "SecureString"
-  value  = "${var.config.bucket_name}.${var.environment}.${var.application}.uktrade.digital"
+  value  = var.environment == "prod" ? "${var.config.bucket_name}.${var.application}.prod.uktrade.digital" : "${var.config.bucket_name}.${var.environment}.${var.application}.uktrade.digital"
```

Comment:
I guess not apart of the fix in this PR but do URLs for prod normally omit the environment completely?

Reply:
I will defer to @ejayesh here.